src/algorithm/hnswlib/StreamWriter.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright 2024-present the vsag project
```

Comment:
code style: add a blank line before this

Reply:
done